specificatie/unresolved/openapi.yaml

```diff
@@ -791,6 +799,14 @@ components:
           type: "string"
         betreftGedeelteVanPerceel:
           type: "boolean"
+        geldverstrekker:
+          type: "object"
+          description: "Dit element is opgenomen om de identificatie van de geldverstrekker beschikbaar te stellen voor partijen die geen gebruik willen maken van de HalLinks voor een verwijzing naar een resource buiten het domein van deze API"
+          properties: 
+            identificatie:
+              type: "string"
+          discriminator:
```

Comment:
Volgens mij is dit niet nodig. Er is geen andere entiteit die hiervan inherit. In de resolved versie zie ik het ook niet voorkomen. Waarschijnlijk weggehaald door de resolver

Reply:
Als ik geen discriminator toevoeg hoe weet de consumer dan om welk type geldverstrekker (of beslaglegger of onderpandgever) het gaat. Dit kunnen ingeschreven personen, kadasterpersonen, of ingeschreven niet-natuurlijk personen of niet-ingeschreven niet-natuurlijk personen zijn. Als ik dat niet doorgeef weet de consumer niet welke resource het hier om gaat.

Reply:
OK. Dan begrijp ik het. Alleen zie ik het niet in de resolved versie. Ik denk dat de resolver dit heeft verwijderd omdat er geen andere elementen hiervan afleid. Ik denk dat je dan of de discriminator property moet weghalen en zelf een type property moet toevoegen of multiple inheritance moet toepassen. Ik denk dat de eerste optie in dit geval de beste is.

Reply:
OK. Dan moet ik er trouwens wel weer een object van maken. Bij een discriminator had dat niet gehoeven, maar dat is een probleem bij resolven. Conclusie: Wel een object een een eigen property toevoegen om het type door te geven.